### DISCUSSION TITLE
EMQX Blocked by Malwarebytes

### DISCUSSION THREAD

**Discussion by neolegacy01** [emqx/emqx#14347] [General]
Hello, is EMQX aware that their URLs are being blocked by Malwarebytes and likely other anti-malware organizations. I discovered this when we had someone randomly being blocked when attempting to login on one of our developer apps which happens to use EMQX MQTT service.

After contacting Malwarebytes they told me that they are blocking all calls to and from emqxcloud.com

They cited this threat: https://www.virustotal.com/gui/domain/k358a192.ala.dedicated.aws.emqxcloud.com/community

Which led to to this entry in the ThreatFox IOC DB: https://threatfox.abuse.ch/ioc/1352097/

Which led to this Cleafy incident: https://www.cleafy.com/cleafy-labs/droidbot-insights-from-a-new-turkish-maas-fraud-operation

Malwarebytes won't remove the block until the issue listed above is resolved. Is EMQX aware that this is happening and that their service is being blocked?

**Comment by id** [comment]
Hi @neolegacy01.

Thanks a lot for the heads up. Our cloud team already took actions to take down malicious deployments, and is analyzing what we can do to prevent this from happening in the future. We will follow up with more details soon.

**Comment by zibuyu1995** [comment]
Hi @neolegacy01 

Thank you very much for your valuable feedback.

The domains mentioned in the article were identified during our threat intelligence analysis in November, and we have already taken action to terminate the associated deployments. These domains are no longer accessible, and the trial users involved have been permanently banned.

As an MQTT cloud service provider, we prioritize privacy and do not analyze or store customer message payloads. While this limits our ability to control customer use cases directly, we are committed to terminating services and cooperating fully whenever we receive verified threat reports.

We will also reach out to the security companies to discuss the removal of restrictions on our main domain.
